src/views/InteractiveMap/CardDialogDesign.vue

```diff
@@ -370,43 +343,6 @@ export default Vue.extend({
     }
   },
   watch: {
-    addReactionSearchQuery(query) {
-      this.addReactionSearchResults = [];
-      if (query === null || query.trim().length === 0) {
-        return;
-      }
-
-      this.isLoadingAddReaction = true;
-      axios
-        .get(
-          `${settings.apis.bigg}/search?query=${query}&search_type=reactions`
-        )
-        .then(response => {
-          // Wait for the full model to be able to filter out results that
-          // already exist in the model.
```

Comment:
We are missing this kind of filtering in mnx. Do we need it?

Reply:
I think it's fine to ignore it, it's not that important to filter them out. Maybe there could be some use case to add a reaction twice now with minor modifications for example.